cli/src/lib/semver.js

```diff
@@ -8,7 +8,7 @@ export type Version = {
   major: number | "x",
```

Comment:
@thejameskyle: It wraps that dependency, not duplicates it. The semver forms that we accept in flow-typed are a limited form of the full-blown types in `semver` -- so we use this module to help ensure we stick to those limited forms.

Additionally, we now have a need to compare 2 ranges (whereas `node-semver` will only compare exact versions to a range)